packages/synapse-react-client/src/components/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/DataAccessRequestAccessorsFilesForm/DataAccessRequestAccessorsFilesForm.tsx

```diff
@@ -145,7 +145,7 @@ export default function DataAccessRequestAccessorsFilesForm(
       // Infinite staleTime ensures this won't get re-fetched unless explicitly invalidated by the mutation
       staleTime: Infinity,
       // This call should never fail, so if we run into an error, throwing to an error boundary is acceptable
-      useErrorBoundary: true,
+      throwOnError: true,
```

Comment:
`useErrorBoundary` was renamed to `throwOnError` so the feature could be used with libraries other than React. The behavior in React is the same.